lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -2694,7 +2694,7 @@ public function InParameter()
             return $this->InputParameter();
         }
 
-        return $this->Literal();
+        return $this->ArithmeticExpression();
```

Comment:
Documentation changes in https://github.com/doctrine/doctrine2/blob/22ecc2d58cdf1e8a9b932d99b952c94afae7e873/docs/en/reference/dql-doctrine-query-language.rst#ebnf are also needed

Reply:
Thanks!